eng/Packages.Data.props

```diff
@@ -337,11 +337,23 @@
     <PackageReference Update="xunit.runner.visualstudio" Version="2.4.2" />
     <PackageReference Update="xunit" Version="2.4.2" />
   </ItemGroup>
-
   <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
     <PackageDownload Update="NETStandard.Library.Ref" Version="[2.1.0]" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsWcfLibrary)' == 'true'">
```

Comment:
I'd prefer that this be explicit with the checks, but I'm not strictly opposed to the current form.   If we keep the short hand, please add a comment attached to this block explaining that it traces back to a specific product set rather than being a general concept that applies to the Azure SDK.

Reply:
Comment was added explaining what this is.